ext/openssl/openssl.c

```diff
@@ -3609,6 +3692,7 @@ static EVP_PKEY *php_openssl_pkey_from_zval(zval *val, int public_key, char *pas
 				key = PEM_read_bio_PrivateKey(in, NULL, php_openssl_pem_password_cb, &password);
 			}
 			BIO_free(in);
+			} // TODO indent, just to please/ease the first round of code review
```

Comment:
Can you resolve this pls?

Reply:
done, https://github.com/php/php-src/pull/6860/commits/27d8f44af90e66dcdb423979a15dc91f4d7f83e4